### DISCUSSION TITLE
Welcome to BenchExec Discussions!

### DISCUSSION THREAD

**Discussion by PhilippWendler** [sosy-lab/benchexec#902] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Naman-Priyadarshi** [comment]
Hello! Naman this side, My primary goal is to learn and gain knowledge from this community and give it back to the community by making contributions! secondary goal being selected as a GSoC 2023 contributor under the project "Modern Tests". Thank you!

**Comment by HarshvMahawar** [comment]
Hello @PhilippWendler, My name is Harsh Vardhan, and going across the GSoC 2023 organizations and their potential projects I found this project "Test framework" most intriguing and approachable. I would like to know how I can improve my understanding of this project. Are there any resources I should go through? or any issues I can try to solve to get familiar with the code base. Thanks

**Reply by PhilippWendler** [reply]
Hi, thank you for contacting us! For general information about BenchExec, I would suggest the Readme and the [linked paper](https://doi.org/10.1007/s10009-017-0469-y) (Chapter 6 is most relevant) and [slides](https://www.sosy-lab.org/research/prs/Latest_ReliableBenchmarking.pdf).

**Comment by chiragbiradar** [comment]
@PhilippWendler @Naman-Priyadarshi @HarshvMahawar @blizzard4591 @hgoes 
Greetings everyone, my name is Chirag Biradar, and I am delighted to be working with all of you. I am highly interested in the BenchExec - Modern Tests GSoC project, and I have thoroughly reviewed the available documentation and slides to gain a better understanding of the project requirements. After careful consideration, I strongly believe that the pytest framework would be the most suitable option for this project.

I am currently in the process of reviewing the codebase to gain a more comprehensive understanding of the project's technical requirements. I am confident that my technical expertise, combined with my passion for this project, will enable me to make valuable contributions to the team.

If any of you have any suggestions or insights regarding the project, I would be grateful to hear them. Thank you for your time, and I look forward to working with you!

**Reply by PhilippWendler** [reply]
Welcome! Nice to hear from you!

**Comment by yashitz07** [comment]
Hii @PhilippWendler I am going through the repos of sosy-lab and found it interesting although I am still a little unfamiliar with some of the tech stack that is used but I want to deep dive into this. Is there any medium where we can interact and have more clarity on this as I am willing to contribute to this.

**Reply by PhilippWendler** [reply]
Hi, and welcome!

For any questions about BenchExec, the GitHub Discussions are the right place.

**Reply by yashitz07** [reply]
Is there any specific beginner issue which I can start with or documentation which i should begin with as I have been reading out some of the issues from repo to get clarity

**Reply by PhilippWendler** [reply]
Our documentation is the one that is checked in, particularly in `doc/`. I think right now we do not have any particular open beginner issues.

**Reply by yashitz07** [reply]
okay got it. Also I want to know whether the organization will be there for GSOC'25. And if it is then is there any specific problem statements on which there should work on?

**Reply by PhilippWendler** [reply]
We do not know yet whether we will be part of GSoC'25 because this depends on whether Google accepts us. But we have some issues labeled as [potential GSoC issues](https://github.com/sosy-lab/benchexec/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen+label%3AGSoC) from previous participations.
